loafwallet.xcodeproj/project.pbxproj

```diff
@@ -3742,7 +3737,7 @@
 			);
 			name = loafwallet;
 			packageProductDependencies = (
-				C30AFB4D2598FC1B00CDCF69 /* SwiftPackageProductDependency */,
```

Comment:
seems like an old xcode thing where the comment used to be less descriptive? My xcode generated it like this. Which should we have?